### ISSUE TITLE
Feature Request: Location/Signature Slur/Profanity filter

### ISSUE THREAD

**Issue by Spikestuff** [adelikat/tasvideos#1808]
Prevent certain words being placed into the Location field.
The issue would having the ability to detect any variant used with numbers, special characters, spaces and full width.
The other issue is due to how open the community is there's also the eventual bigots (I write "eventual" even though there's an instance of a father that basically wants no association with any progressive site), so I guess string words as well.

**Comment by bigbass1997** [comment]
After discussing with the moderation team, the majority were against this feature.

The idea has merit for sure, but these filters rarely work to actually prevent bad speech. If someone wants to be racist, they're gonna be racist. Trying to come up with an extensive and robust filter list would be challenging and ultimately may cause more harm than good (e.g. false-positives and technical debt).

My personal issue is that even if the filter worked, it only masks a user's behavior. If there's a bigot, racist, or sexist among us, I'd rather they expose themselves for who they are so we can remove them from the community altogether.

An alternative _might_ be more like a detection feature that notifies moderators when a certain word is used, but that'd need more discussion.

I think this issue can be closed.

**Comment by Spikestuff** [comment]
Yeah this was mainly made as a hypothetical back then or something, just check what I wrote in either area around this time and you'd probably find the actual reasoning on why this issue request existed in the first place. 👍
